src/index.js

```diff
@@ -0,0 +1,233 @@
+        './assets/weather-imgs/weather_4.jpg',
+        './assets/weather-imgs/weather_5.jpg'
+    ];
+    let currentIndex = 0;
+
+    // rotates weather album images
+    const rotateImage = () => {
+        currentIndex = (currentIndex + 1) % images.length;
+        landscapeImg.style.backgroundImage = `url(${images[currentIndex]})`;
+    };
+
+    setInterval(rotateImage, 10000);
+};
+
+// reset city name section
+const updateCityNameReset = () => {
+    // resets city name and temperature
+    const resetCityInfo = () => {
+        cityNameHeader.textContent = defaultCityNameInput;
+        currentTemperature.value = defaultTempValue
+        tempValue.textContent = displayTemperature();
+        tempValue.style.color = "red";
+        cityNameTextBox.value = "";
+        updateWeatherDisplay();
+        updateBackground();
+    };
+
+    resetCityName.addEventListener("click", resetCityInfo);
+};
+
+// Get Temp of search terms
+const tempSearch = async (cityName) => {
+    const locationResponse = await axios.get(`${baseURL}/location`, {
+        params: {q: cityName}
+    });
+    const {lat, lon} = locationResponse.data[0];
+
+    const weatherResponse = await axios.get(`${baseURL}/weather`, {
+        params: {lat, lon}
+    });
+    currentTemperature.value = weatherResponse.data.main.temp;
+    tempValue.textContent = displayTemperature();
+    updateWeatherDisplay();
+};
+
+// - Changing the temperature should change the page's background.
+const updateBackground = () => {
+    const body = document.body;
+    let temp = kelvinToF(currentTemperature.value);
+    if (temp > 80) {
+        currentTemperature.image = "hot.png"
+    } else if (temp > 65) {
+        currentTemperature.image = "good.png"
+    } else {
+        currentTemperature.image = "cold.png"
+    }
+    body.style.backgroundImage = `url(./assets/tempretures/${currentTemperature.image})`;
+    body.style.backgroundSize = "cover";
+};
+
+// sky display
+
+const updateSkyDisplay = (selectedSky) => {
+    const skyDisplay = document.getElementById("sky");
+    skyDisplay.innerHTML = skies[selectedSky] || skies.chooseAForecast;
+};
+
+skySelect.addEventListener("change", (event) => {
+    const selectedSky = event.target.value;
+    const sky = document.getElementById("sky");
+    sky.innerHTML = "";
+    sky.textContent = skies[selectedSky];
+    updateSkyDisplay(selectedSky);
+});
+
+
+// set temp control
+increaseTempControl.addEventListener("click", () => {
+    if(currentTemperature.unit === "F"){
+        currentTemperature.value +=  0.555;
+    } else {
+        currentTemperature.value += 1;
+    }
+    tempValue.textContent = displayTemperature();
+    updateWeatherDisplay();
+});
+decreaseTempControl.addEventListener("click", () => {
+    if(currentTemperature.unit === "F") {
+        currentTemperature.value -=  0.555;
+    } else {
+        currentTemperature.value -= 1;
+    }
+    tempValue.textContent = displayTemperature();
+    updateWeatherDisplay();
```

Comment:
Same as above, prefer this in a helper function

Reply:
Notice that the difference between these two blocks of code is `+=` and `-=`.

Consider combining this logic into one helper function `handleTempChange` that could be registered with both the ⬆️ and ⬇️ buttons.

```javascript
const handleTempChange = (event) => {
    if (event.target.id === 'increaseTempControl') {
       // increase the temp
    } else {
       //decrease the temp
    }
  updateWeatherDisplay();
};